codice/ddf
# [2.26.x] Respect the GeometryOperands from a WFS 1.1 GetCapabilities

#### What does this PR do?
The GeometryOperands were hardcoded in the WFS 1.1 Source. Now the GeometryOperands from the GetCapabilities will be used to determine if a operand is supported. There is a new setting available to override this change and use the old behavior of supporting all geometry operands.

#### Who is reviewing it? 

@derekwilhelm 

#### Select relevant component teams: 

@codice/ogc 

#### Ask 2 committers to review/merge the PR and tag them here.

@jlcsmith

#### How should this be tested?


- Configure a WFS 1.1 source using a Geoserver WFS 1.1.0 service (by default, only Envelope, Point, LineString, and Polygon should be supported).
- Query with a MultiGeometry, MultiLineString, or MultiPoint and verify it is not supported
- Query with a MultiGeometry and verify it falls back to Polygon (one polygon contained) or Envelope (multiple polygons contained)
- Configure the WFS 1.1 source to force all geometry operands
- Requery with a MultiGeometry, MultiLineString, or MultiPoint and verify it is sent to Geoserver

#### Any background context you want to provide?

#### What are the relevant tickets?
Fixes: n/a

#### Screenshots


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
